src/renderer/components/cluster-manager/lens-views.ts

```diff
@@ -42,9 +41,9 @@ export async function autoCleanOnRemove(clusterId: ClusterId, iframe: HTMLIFrame
   // Keep frame in DOM to avoid possible bugs when same cluster re-created after being removed.
   // In that case for some reasons `webFrame.routingId` returns some previous frameId (usage in app.tsx)
   // Issue: https://github.com/lensapp/lens/issues/811
-  iframe.dataset.meta = `${iframe.name} was removed at ${new Date().toLocaleString()}`;
-  iframe.removeAttribute("src")
+  iframe.dataset.meta = `${iframe.name} was removed at ${new Date().toLocaleString()}`
   iframe.removeAttribute("name")
+  iframe.contentWindow.postMessage("teardown", "*")
```

Comment:
```suggestion
  iframe.contentWindow.postMessage("teardown", getClusterFrameUrl(clusterId))
```
but in `getClusterFrameUrl` should be added `location.protocol` before `//` explicitly (required by postMessage)

Reply:
Why do we need this? Context is already an iframe so it cannot go anywhere else.

Reply:
This is saying from docs:

>  Always provide a specific targetOrigin, not *, if you know where the other window's document should be located. Failing to provide a specific target discloses the data you send to any interested malicious site.

Maybe it's more safe.. otherwise i don't know why specifiing `targetOrigin` needed to `postMessage` at all.